_includes/header.html

```diff
@@ -10,7 +10,7 @@
 </script>
 <div align="center">
     <br />
-    <img alt="Strimzi" src="assets/strimzi.png" />
+    <img alt="Strimzi" src="https://strimzi.io/assets/strimzi.png" />
```

Comment:
isn't it enough just `/assets/strimzi.png`?

Reply:
Might ... but maybe that will bubble out somwehere else. This sjhould be bulletproof :-)

Reply:
Why not trying then? we have this cool preview feature thanks to netlify :-)

Reply:
I should have left you do it. We had the preview before as well ... and how much did it helped?

Reply:
I forgot about it before :P